### ISSUE TITLE
Language button label not being read by NVDA

### ISSUE THREAD

**Issue by jhung** [platform-coop-toolkit/pinecone#174]
**Describe the bug**
The Language button label is not being read by NVDA. NVDA reports: "Button collapsed", but does not read the label. Did not test with other screen readers.

**To Reproduce**
1. Launch a screen reader
2. Go to the Menu molecule.
3. Move focus until Language item is focused.
4. Notice that the screen reader does not report the label of the button.

**Expected behavior**
Screen reader should read the label of the button when focused.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox 72.0.1
- Screenreader: NVDA 2019.2.1

**Additional context**
Other screen readers may exhibit this issue, but was only tested on NVDA.
